contents/blog/posthog-vs-kubit.md

```diff
@@ -0,0 +1,412 @@
+            <td>Receive regular alerts about new data</td>
+        </tr>
+        <tr>
+            <td><strong>Embeddable dashboards</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Embed dashboards in other pages</td>
+        </tr>
+        <tr>
+            <td><strong>Slack integration</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>Discord integration</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>MS Teams integration</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>Data management suite</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Categorize and define a list of events and properties</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> ### Should you use autocapture?
+> 
+> Autocapturing events is a controversial topic for data teams. The argument against it is that it introduces more noise and can introduce additional costs. The argument for it is that it collects more data for users to collaborate with, at greater consistency. 
+>
+> At PostHog, [we think it’s better to have more information](/blog/is-autocapture-still-bad), than less. We give users their first 1M events for free every month too, to negate the costs argument. But if you disagree, autocapture can be easily disabled. 
+
+## Library support
+PostHog supports a wide range of client and server libraries, but not all features are equally available across all of them. We recommend using PostHog's JavaScript snippet to enjoy all our features. See our [client library documentation](/docs/getting-started/install?tab=snippet) for more information.
+
+Kubit, on the other hand, claims to be a no-code product without any SDKs or libraries. The platform is closed source, so it’s impossible to verify how this works as [the documentation gives](https://docs.kubit.ai/docs) no information. Instead, Kubit says it only ‘talks directly to your data warehouse’.
```

Comment:
I think we should tone down this language. Generally, these comparisons should be about facts, rather than our opinion on the solution. The reader can figure out for themselves what solution is better for them.

Reply:
I get that this is potentially a bit snarky, but it's 100% based on facts and isn't opinion. Same for the comment below: Contacted Sales, yet to hear back.

We could soften to the below, but anymore and we're starting to both omit a relevant weakness of the product and lose the opinionated tone we strive for elsewhere. 

```suggestion
Kubit, on the other hand, is a closed-source, no-code product and [the documentation](https://docs.kubit.ai/docs) doesn't offer information about SDKs or libraries. 
```

Reply:
1. RE: sales, it's an enterprise company. I imagine they triage incoming queries and respond to the ones that fir their our ICP. We do exactly the same. It also just feels sneaky digging them out on it.

2. RE: suggestion, now we're just omitting the fact they talk directly to your data warehouse. Surely the point is they don't have SDKs / libraries because their product is only built to work on top of an existing data warehouse?